python/ray/train/v2/api/data_parallel_trainer.py

```diff
@@ -4,8 +4,15 @@
 import ray
 from ray._private.ray_constants import env_bool
 from ray.air._internal.usage import tag_train_v2_trainer
-from ray.train import BackendConfig, Checkpoint
-from ray.train._internal.data_config import DataConfig
+from ray.train import (
+    BackendConfig,
+    Checkpoint,
+    DataConfig,
+    ScalingConfig,
+    RunConfig,
+    Result,
+    UserCallback,
+)
```

Comment:
This is so that the "module" of these configs stays consistent. Ex: `ray.train` instead of `ray.train.v2.api.config`. Might also be needed for doc api refs too.